sspl_test/alerts/os/test_systemd_service_actuator.py

```diff
@@ -148,6 +150,7 @@ def service_actuator_request(service_name, action):
                         "node_id": "1"
                     },
                     "response_dest": {},
+                    "target_node_id": node_id,
```

Comment:
so this code tests actuator request on same now. which is fine

Reply:
Yes, we will need to make changes here later as per the 3-node sanity plan.